responder_queue.go

```diff
@@ -59,7 +62,18 @@ func newResponderQueue(capacity int) *responderQueue {
 func (e *responderQueue) Forward(r *Robot, ch <-chan Message) {
 	for msg := range ch {
 		rs := newResponder(r, msg)
+
+		isDirectMessage := false
+		slackMessage, ok := msg.Envelope.(slack.Message)
```

Comment:
🤔 this feels like something the slack adapter should handle instead

Reply:
Sorry, what do you mean by slack adapter?

Reply:
In case someone else comes across this comment and is confused: https://github.com/botopolis/slack